stdlib/camlinternalAtomic.ml

```diff
@@ -2,7 +2,8 @@
 (*                                                                        *)
 (*                                 OCaml                                  *)
 (*                                                                        *)
-(*             Gabriel Scherer, projet Partout, INRIA Paris-Saclay        *)
+(*                 Stephen Dolan, University of Cambridge                 *)
+(*          Guillaume Munch-Maccagnoni, projet Gallinette, INRIA          *)
 (*                                                                        *)
 (*   Copyright 2020 Institut National de Recherche en Informatique et     *)
 (*     en Automatique.                                                    *)
```

Comment:
Should probably add the copyright of Cambridge and/or Stephen Dolan.
cc @stedolan

Reply:
You can also remove mine which I think concerned the non-multicore emulation.

Note that there is a redundant comment a few line below that.